plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/StandardDeployPreferencesPanel.java

```diff
@@ -93,11 +93,14 @@
   private DataBindingContext bindingContext;
 
   private Runnable layoutChangedHandler;
+  private boolean childOfPropertyDialog = false;
```

Comment:
The name childOfPropertyDialog seems rather specific. Maybe allowEmptyProjectId?